CMakeLists.txt

```diff
@@ -451,6 +449,19 @@ target_include_directories(ocos_operators PRIVATE ${GSL_INCLUDE_DIR})
 list(APPEND ocos_libraries Microsoft.GSL::GSL)
 
 list(REMOVE_DUPLICATES OCOS_COMPILE_DEFINITIONS)
+target_compile_definitions(noexcep_operators PRIVATE ${OCOS_COMPILE_DEFINITIONS})
+if(NOT OCOS_ENABLE_CPP_EXCEPTIONS)
+  if(MSVC)
+    get_target_property(_target_cxx_flags noexcep_operators COMPILE_OPTIONS)
+    list(REMOVE_ITEM _target_cxx_flags "/EHsc")
+    list(APPEND _target_cxx_flags "/EHs-c-")
```

Comment:
out of interest is there an advantage to doing REMOVE + APPEND instead of REPLACE?

Reply:
REPLACE requires regular-expression, but I am not very familiar it and think it may need some escaping char for the flag.